### PR TITLE
Properly check return value from Set and Call

### PR DESCRIPTION
**Pull request by LinusU**
Thanks for contributing!

- [x] Have you updated CHANGELOG.md?

### CODE REVIEW

**Comment by LinusU** [comment]
Blocked on https://github.com/nodejs/nan/issues/851

File: src/Backends.cc [hunk 1]
```diff
@@ -14,6 +14,5 @@ void Backends::Initialize(Local<Object> target) {
   PdfBackend::Initialize(obj);
   SvgBackend::Initialize(obj);
 
-  Local<Context> v8ctx = Nan::GetCurrentContext();
-  target->Set(v8ctx, Nan::New<String>("Backends").ToLocalChecked(), obj);
+  Nan::Set(target, Nan::New<String>("Backends").ToLocalChecked(), obj).ToChecked();
```
Comment (LinusU) [inline src/Backends.cc]: todo: replace `ToChecked()` call that doesn't use the return value with `Check()`

**Comment by zbjornson** [comment]
I think this can land now with Nan 2.14.0. :shipit:

File: src/Backends.cc (hunk 1 above)
Comment (LinusU) [reply src/Backends.cc]: Fixed 👌

**Comment by LinusU** [comment]
Rebased on master and upgraded nan 🚀

**Pull request merged**